test/e2e/network_segmentation_services.go

```diff
@@ -233,13 +233,16 @@ var _ = Describe("Network Segmentation: services", func() {
 				Expect(err).NotTo(HaveOccurred())
 
 				By("Verify the UDN client connection to the default network service")
-				checkNoConnectionToClusterIPs(f, udnClientPod2, defaultService)
-				// TODO uncomment below when below OVN_DISABLE_SNAT_MULTIPLE_GWS=true is supported
-				// checkConnectionToNodePort(f, udnClientPod2, defaultService, &nodes.Items[0], "server node", defaultServerPod.Name)
-				// TODO change line below to checkConnectionToNodePort when we have full UDN support in ovnkube-node
-				checkNoConnectionToNodePort(f, udnClientPod2, defaultService, &nodes.Items[1], "local node")
-				// TODO uncomment below when OVN_DISABLE_SNAT_MULTIPLE_GWS=true is supported
-				// checkConnectionToNodePort(f, udnClientPod2, defaultService, &nodes.Items[2], "other node", defaultServerPod.Name)
+				// FIXME(tssurya): https://github.com/ovn-org/ovn-kubernetes/issues/4687
+				if !isLocalGWModeEnabled() {
+					checkNoConnectionToClusterIPs(f, udnClientPod2, defaultService)
+					// TODO uncomment below when below OVN_DISABLE_SNAT_MULTIPLE_GWS=true is supported
```

Comment:
isn't this supported now?

Reply:
yup will remove this

Reply:
good catch.
done.

Reply:
hmm there is no defaultServerPod defined.. talking to ric to see what needs to be done here.